implementations/elixir/ockam/ockam_hub/lib/hub/service/provider.ex

```diff
@@ -73,7 +73,7 @@ defmodule Ockam.Hub.Service.Provider do
     service_providers_map = get_service_providers_map(providers)
 
     Enum.map(services_config, fn service_config ->
-      do_get_service_child_spec(service_config, service_providers_map)
+      handle_get_service_child_spec(service_config, service_providers_map)
```

Comment:
Is this change on purpose or just a part of some rename?
Seems like some `do_` were auto-renamed to `handle_`

Reply:
my bad